test/old/testdir/test_functions.vim

```diff
@@ -2104,7 +2104,7 @@ func Test_trim()
   call assert_equal(" \tabcd\t     xxxx   tail", trim(" \tabcd\t     xxxx   tail", "abx"))
   call assert_equal("RESERVE", trim("你RESERVE好", "你好"))
   call assert_equal("您R E SER V E早", trim("你好您R E SER V E早好你你", "你好"))
-  call assert_equal("你好您R E SER V E早好你你", trim(" \n\r\r   你好您R E SER V E早好你你    \t  \x0B", ))
+  call assert_equal("你好您R E SER V E早好你你", trim(" \n\r\r   你好您R E SER V E早好你你    \t  \x0B"))
```

Comment:
?

Reply:
Not sure about this. Maybe stylua or sed did it.

Reply:
The dog did it

Reply:
Interesting that ci didn't fail. Must be a vimscript problem.

Reply:
Interesting that ci didn't fail. Must be a vimscript problem.

Reply:
Will revert his in #27024